### PR TITLE
move secret into swarm thing in swagger

### PR DESCRIPTION
**Pull request by allencloud**
Signed-off-by: allencloud <allen.sun@daocloud.io>

**- What I did**
1. move secret into swarm thing in swagger

ping @thaJeztah @vdemeester

### CODE REVIEW

**Comment by vieux** [comment]
ping @dnephin

File: api/swagger.yaml [hunk 1]
```diff
@@ -137,6 +135,10 @@ tags:
     x-displayName: "Tasks"
     description: |
       A task is a container running on a swarm. It is the atomic scheduling unit of swarm. Swarm mode must be enabled for these endpoints to work.
+  - name: "Secret"
+    x-displayName: "Secrets"
+    description: |
+      Secrets are the definitions of secrets to run on a swarm. Swarm mode must be enabled for these endpoints to work.
```
Comment (dnephin) [inline api/swagger.yaml]: How about "Secrets are sensitive data that can be used by services..."

Comment (allencloud) [inline api/swagger.yaml]: Yeah, that makes sense. Update. Thanks @dnephin

**Review by vdemeester — APPROVED** [review]
LGTM 🐮

**Review by dnephin — APPROVED** [review]
LGTM

**Comment by thaJeztah** [comment]
cherry-picked into 17.03.1 through https://github.com/docker/docker/pull/31754

**Pull request merged**